modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -25,7 +26,9 @@ const defaultProps = {
 export default class MVTLayer extends TileLayer {
   initializeState() {
     super.initializeState();
+    const binary = this.context.viewport instanceof GlobeViewport ? false : this.props.binary;
```

Comment:
Use `viewport.resolution`. Layers should not depend on a particular viewport constructor.